extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -119,6 +124,16 @@ export class DataTierApplicationWizard {
 		this.configureButtons();
 
 		this.wizard.open();
+
+		// Data tier wizard cancel button click event
+		this.wizard.cancelButton.onClick(() => {
```

Comment:
The wizards should have telemetry already for the built-in buttons like cancel. Did you see if this was the case? If it isn't then we should be doing that so that we don't have every extension needing to duplicate this kind of logic.

https://github.com/microsoft/azuredatastudio/commit/807a4ae8c448ba97caa4b995aa4a278e8a4361e8

Although we don't keep track of the page that was closed - but that seems like a good idea so I'd suggest adding that to the wizard telemetry I already have added.

Reply:
Good one, I have added the current page title and the wizard title to the telemetry on cancel/esc button event. Thanks 👍